low-level/framework/messaging/ingress_processor.py

```diff
@@ -149,7 +149,7 @@ def _process_msg(self, body):
 
         ingressMsg = {}
         uuid = None
-        hostname = socket.getfqdn()
+        hostname = socket.gethostname()
```

Comment:
so check for hostname already exists, which is serving this usecase already

Reply:
Sorry @cdeshmukh, I didn't get you, what do you mean by hostname already exists?